Assets/MixedRealityToolkit/Utilities/StandardShader/HoverLight.cs

```diff
@@ -26,22 +26,30 @@ public class HoverLight : MonoBehaviour
         /// <summary>
         /// Specifies the Radius of the HoverLight effect
```

Comment:
Please make sure to keep the two summary sections next to the public properties.